### PR TITLE
Hash only

### PR DESCRIPTION
**Pull request by pbstriker38**

### CODE REVIEW

File: src/utils/hash.ts [hunk 1]
```diff
@@ -40,6 +41,16 @@ export async function hashValues(payload: any, jwt: string) {
   return result;
 }
 
+export async function hashValue(value: string) {
```
Comment (mmclead) [inline src/utils/hash.ts:44]: can we reuse this function inside the hashAndEncryptValues function?

File: test/events/git_event.test.ts [hunk 2]
```diff
@@ -83,5 +87,9 @@ describe("Test git_event functions", function () {
       const fileChangeContext: any = contexts.find((n: any) => n.schema.includes("file_change"));
       expect(Object.keys(fileChangeContext.data)).to.eql(['file_name', 'insertions','deletions']);
     });
+
+    it("does not encrypt the file_change file_names", async function() {
+
```
Comment (mmclead) [inline test/events/git_event.test.ts:92]: test?

File: src/utils/hash.ts (hunk 1 above)
Comment (pbstriker38) [reply src/utils/hash.ts:44]: Definitely. But we'll be doing that nasty sodium ready check in the loop.

Comment (mmclead) [reply src/utils/hash.ts:44]: right.   I did see that.

File: test/events/git_event.test.ts (hunk 2 above)
Comment (pbstriker38) [reply test/events/git_event.test.ts:92]: blah!!! I started refactoring and completely forgot 🤦

Comment (pbstriker38) [reply test/events/git_event.test.ts:114]: added

**Review by mmclead — APPROVED** [review]

**Pull request merged**